Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN make dist/clusterrunner
 
 # STAGE 2: CentOS 7 base w/ fpm to package pex into an rpm.
 #          NOTE: To update image run: "docker pull cdrx/fpm-centos:7"
-FROM cdrx/fpm-centos@sha256:fd314b5b8fdc78f714ba6d3e83d45ae58dd8dfff7f21320e55ec854587b01c2f AS stage2
+FROM cdrx/fpm-centos:7 AS stage2
```

Comment:
Is this pinned to a version?

Reply:
No, Pinning doesn't work unfortunately because the remotes don't keep old versions. I guess to implement pinning we need to have an docker image cache.

I guess I can remove the comment above this change.